pkgs/by-name/re/re-isearch/package.nix

```diff
@@ -7,15 +7,15 @@
   libnsl,
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttr: {
   pname = "re-Isearch";
-  version = "unstable-2022-03-24";
+  version = "2.20220925.4.0a-unstable-2025-03-16";
 
   src = fetchFromGitHub {
     owner = "re-Isearch";
     repo = "re-Isearch";
-    rev = "e5953ea6c84285283be3689df7065908369cdbaf";
-    sha256 = "sha256-D0PDqlWzIOHqdS2MlNzR2T5cyhiLcFlf30v6eFokoRQ=";
+    rev = "56e0dfbe7468881b3958ca8e630f41a5354e9873";
+    sha256 = "sha256-tI75D02/sFEkHDQX/BpDlu24WNP6Qh9G0MIfEvs8npM=";
   };
 
```

Comment:
Can't leave comments on unchanged lines, but these 2 lines need to be added to the `preBuild` phase:

```diff
  preBuild = ''
    cd build
+   export HOME="$TMPDIR" # wants to write to HOME
+   make clean # clean up pre-built objects in the source
    makeFlagsArray+=(
      EXTRA_INC="-I${db.dev}/include -I${lib.getDev file}/include"
      LD_PATH="-L../lib -L${db.out}/lib -L${file}/lib -L${libnsl}/lib"
    )
  '';
```